Dependency in archetype does not resolve when postinstalling as dependency

## Overview

`npm install` in [`victory-voronoi`](https://github.com/FormidableLabs/victory-voronoi) fails.

`victory-voronoi` depends on `victory-animation`, which runs a [postinstall script](https://github.com/FormidableLabs/victory-animation/blob/master/package.json#L17): `builder run npm:postinstall`, which fails because `rimraf` can't be found.

`victory-animation` uses the `builder-react-component` archetype, which `rimraf` [is part of](https://github.com/FormidableLabs/builder-react-component/blob/master/package.json#L63).

`rimraf` also happens to be a dependency of `victory-voronoi`, but I assume it isn't installed yet at the time `victory-animation` runs its' postinstall script.


### NPM and Node versions

```
$ npm -v
2.14.7
$ node -v
v4.2.2
```

### NPM debug output
```
> victory-animation@0.0.6 postinstall /Users/per/dev/formidable/victory-voronoi/node_modules/victory-animation
> npm run build-lib

|
> victory-animation@0.0.6 build-lib /Users/per/dev/formidable/victory-voronoi/node_modules/victory-animation
> npm run clean-lib && babel --stage 0 src -d lib

/
> victory-animation@0.0.6 clean-lib /Users/per/dev/formidable/victory-voronoi/node_modules/victory-animation
> rimraf lib

sh: rimraf: command not found

npm ERR! Darwin 15.0.0
npm ERR! argv "/Users/per/.nvm/versions/v4.2.2/bin/node" "/Users/per/.nvm/versions/v4.2.2/bin/npm" "run" "clean-lib"
npm ERR! node v4.2.2
npm ERR! npm  v2.14.7
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! victory-animation@0.0.6 clean-lib: `rimraf lib`
npm ERR! spawn ENOENT
npm ERR!
npm ERR! Failed at the victory-animation@0.0.6 clean-lib script 'rimraf lib'.
npm ERR! This is most likely a problem with the victory-animation package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     rimraf lib
```

/cc @exogen @ryan-roemer

--- a/package.json
+++ b/package.json
@@ -14,7 +14,7 @@
   },
   "homepage": "https://github.com/FormidableLabs/victory-voronoi",
   "scripts": {
-    "postinstall": "npm run build-lib",
+    "postinstall": "node -e \"require('fs').stat('lib', function(e,s){process.exit(e || !s.isDirectory() ? 1 : 0)})\" || npm run build-lib",
     "preversion": "npm run check",
     "version": "npm run clean && npm run build && git add -A dist",
     "clean-dist": "rimraf dist",

--- a/package.json
+++ b/package.json
@@ -14,7 +14,7 @@
   },
   "homepage": "https://github.com/FormidableLabs/victory-voronoi",
   "scripts": {
-    "postinstall": "node -e \"require('fs').stat('lib', function(e,s){process.exit(e || !s.isDirectory() ? 1 : 0)})\" || npm run build-lib",
+    "postinstall": "cd lib || npm run build-lib",
     "preversion": "npm run check",
     "version": "npm run clean && npm run build && git add -A dist",
     "clean-dist": "rimraf dist",

--- a/package.json
+++ b/package.json
@@ -63,7 +63,7 @@
     "webpack": "^1.10.0"
   },
   "devDependencies": {
-    "babel-eslint": "^3.1.25",
+    "babel-eslint": "^5.0.0",
     "chai": "^3.2.0",
     "eslint": "^0.24.1",
     "eslint-config-defaults": "^3.0.3",

